src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Namespace/NewAzureServiceBusNamespace.cs

```diff
@@ -85,7 +91,14 @@ public override void ExecuteCmdlet()
             {
                 try
                 {
-                    WriteObject(Client.BeginCreateNamespace(ResourceGroupName, Name, Location, SkuName, tagDictionary, SkuCapacity));
+                    if (EnableZoneRedundant.IsPresent)
+                    {
+                        WriteObject(Client.BeginCreateNamespace(ResourceGroupName, Name, Location, SkuName, tagDictionary, SkuCapacity, EnableZoneRedundant));
+                    }
+                    else
+                    {
+                        WriteObject(Client.BeginCreateNamespace(ResourceGroupName, Name, Location, SkuName, tagDictionary, SkuCapacity, EnableZoneRedundant));
```

Comment:
@v-Ajnava it looks like this call is the same as the one above -- was this intentional?

Reply:
my bad, else one should not have 'EnableZoneRedundant', corrected.